root/static/scripts/edit/externalLinks.js

```diff
@@ -44,6 +43,7 @@ type LinkStateT = {
 };
 
 type LinkHashT = {
+  __proto__: empty,
```

Comment:
What's this for?

Reply:
It's possible to create objects that have a null prototype with `Object.create(null)`. That's useful if you're using them like Maps, because they won't have any conflicting properties in the prototype chain (methods like `hasOwnProperty` for example). But Flow doesn't like one object type has `__proto__: null` and another doesn't, so I'm typing it as `empty` instead to make it an error to use in any case, which Flow seems to be happy with.

Reply:
Usually you'd use `new Map` to create a proper map, but `Object.create(null)` can be necessary if it's more convenient to use the map like a plain object. (Not sure if that's the case here, but I didn't want to refactor anything too much.)